ossdbs/api.py

```diff
@@ -619,3 +631,17 @@ def run_PAM(settings):
             scaling=settings["Scaling"],
             scaling_index=settings["ScalingIndex"],
         )
+
+
+def _set_edge_hp_flag(electrode: ElectrodeModel, edge_sizes: dict) -> None:
```

Comment:
Move to electrode class as mentioned above

Reply:
Done